codecov.yaml

```diff
@@ -1,5 +1,4 @@
 ignore:
-  - "intertidal/validation.py"
```

Comment:
Stop ignoring validation code as we now cover some of it in our validation tests